renderers/chord_hover.js

```diff
@@ -3,19 +3,103 @@
 const chordDiagram = require('./chord_diagram.js');
 const { guitarChordLibrary } = require('./chord_library.js');
 
+const slowScrollFunction = `
+function slowScroll(id, direction, distance, step) {
+  var element = document.getElementById(id);
+  
+  if (element) {
+    var scrollAmount = 0;
+    var slideTimer = setInterval(function() {
+      element.scrollLeft += direction === 'left' ? -step : step;
+
+      scrollAmount += step;
+      if(scrollAmount >= distance){
+          window.clearInterval(slideTimer);
+      }
+    });
+  }
+}`;
+
+const incrementIdFunction = `
+function incrementId(id, direction) {
+  var countId = id + '-count'
+  var element = document.getElementById(countId);
+
+  if (element) {
+    var values = element.innerHTML.split(' of ');
+    var currentValue = parseInt(values[0]);
+
+    if (direction === 'left' && currentValue > 1) {
+      element.innerHTML = currentValue - 1;
+      element.innerHTML += ' of ' + values[1];
+    } else if (direction === 'right' && currentValue < parseInt(values[1])) {
+      element.innerHTML = currentValue + 1;
+      element.innerHTML += ' of ' + values[1];
+    }
+  }
+}
+`;
+
+function createDiagramDiv(voicing, chordDiagramContainerDiv) {
+  const svg = chordDiagram.renderChordDiagram(voicing);
+
+  const chordDiagramDiv = document.createElement('div');
+
+  chordDiagramDiv.className = 'diagram';
+  chordDiagramDiv.innerHTML += svg;
+
+  chordDiagramContainerDiv.appendChild(chordDiagramDiv);
+}
+
 function addChordToDiv(voiceDiv, chord) {
   voiceDiv.className += ' chord';
 
   if (guitarChordLibrary.has(chord.toString())) {
-    const chordDiagramDiv = document.createElement('div');
-    chordDiagramDiv.className = 'diagram';
+    const containerDiv = document.createElement('div');
+    containerDiv.className = 'diagram-container';
+
+    const contentDiv = document.createElement('div');
+    contentDiv.className = 'diagram-content-container';
+
+    const id = 'chord' + Math.floor(Math.random() * 10000);
```

Comment:
You can also just create a global variable that you increment.

Reply:
This is, btw, an instance of the birthday paradox. If we had 100 chords on the page, the probability of collision is  1% ~ probably a lot higher than you were expecting: 1-(9999/10000)^(100) = 0.01

Reply:
A global variable seems much more reasonable for the id XD

Huh, yeah, that is a lot higher than I assumed! Good to know!

Reply:
Also, I didn't read your original comment regarding whether scrolling causes all corresponding chords to scroll as well. TBH, I hadn't thought about it, but I would say that if you wanted to play v2 of an Esus4, you probably want to do it for all Esus4s, so maybe this logic isn't necessary at all.

Reply:
Missed this comment before merging.

Yeah, I really wasn't sure if there would be a case where you'd need different fingerings at different points. I'll make a task to switch it to scroll all of the same chords.